Themes/default/Profile.template.php

```diff
@@ -157,19 +157,26 @@ function template_summary()
 	// Are there any custom profile fields for above the name?
 	if (!empty($context['print_custom_fields']['above_member']))
 	{
-		echo '
+		$above = '
```

Comment:
Instead of defining $above here and then echoing it later, it'd be cleaner to move the original echo statement down to where you've put `echo $above;`